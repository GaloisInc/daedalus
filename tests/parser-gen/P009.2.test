P009.ddl
--gen
--interp=inputs/P009.2.inp
