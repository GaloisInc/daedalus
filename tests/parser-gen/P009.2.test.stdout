context:
  "Main"
input: File context:
Length: 14 (0xe) bytes
0000:   41 42 43 44  4b 61 62 61  45 46 73 74  [1;91m45[0m 6c   ABCDKabaEFst[1;91mE[0ml

